Description: Do not use build time in docs
 This makes the package reproducible.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-06-06

--- python-osprofiler-1.3.0.orig/doc/source/conf.py
+++ python-osprofiler-1.3.0/doc/source/conf.py
@@ -137,7 +137,8 @@ try:
     html_last_updated_fmt = subprocess.Popen(
         git_cmd, stdout=subprocess.PIPE).communicate()[0]
 except Exception:
-    html_last_updated_fmt = time.ctime()
+    warnings.warn('Cannot get last updated time from git repository. '
+                  'Not setting "html_last_updated_fmt".')
 
 # If true, SmartyPants will be used to convert quotes and dashes to
 # typographically correct entities.
